virt/release_notes/virt-4-15-release-notes.adoc

```diff
@@ -190,73 +197,71 @@ link:https://access.redhat.com/support/offerings/techpreview[Technology Preview
 [discrete]
 [id="virt-4-15-ki-monitoring"]
 ==== Monitoring
-//4.14 Leave in per Stu (fix deferring to 4.15)
-* The Pod Disruption Budget (PDB) prevents pod disruptions for migratable virtual machine images. If the PDB detects pod disruption, then `openshift-monitoring` sends a `PodDisruptionBudgetAtLimit` alert every 60 minutes for virtual machine images that use the `LiveMigrate` eviction strategy. (link:https://bugzilla.redhat.com/show_bug.cgi?id=2026733[*BZ#2026733*])
+//fix targeted for 4.16
+* The Pod Disruption Budget (PDB) prevents pod disruptions for migratable virtual machine images. If the PDB detects pod disruption, then `openshift-monitoring` sends a `PodDisruptionBudgetAtLimit` alert every 60 minutes for virtual machine images that use the `LiveMigrate` eviction strategy. (link:https://issues.redhat.com/browse/CNV-33834[*CNV-33834*])
 ** As a workaround, xref:../../monitoring/managing-alerts.adoc#silencing-alerts_managing-alerts[silence alerts].
 
-[discrete]
-[id="virt-4-15-ki-networking"]
-==== Networking
-//CNV-10590 - This issue can be removed for 4.15 when CNV-13680 RN is added.
-//BZ-1885605
-//4.14 leave in per Miguel Duarte de Mora Barroso
-* If your {product-title} cluster uses OVN-Kubernetes as the default Container Network Interface (CNI) provider, you cannot attach a Linux bridge or bonding device to a host's default interface because of a change in the host network topology of OVN-Kubernetes. (link:https://bugzilla.redhat.com/show_bug.cgi?id=1885605[*BZ#1885605*])
-** As a workaround, you can use a secondary network interface connected to your host, or switch to the OpenShift SDN default CNI provider.
 
-//CNV-33789: Cannot SSH into VM over NodePort and Console's FQDN when using OVNKubernetes networking
-* You cannot SSH into a VM when using the `networkType: OVNKubernetes` option in your `install-config.yaml` file. (link:https://bugzilla.redhat.com/show_bug.cgi?id=2165895[*BZ#2165895*])
+//[discrete]
+//[id="virt-4-15-ki-networking"]
+//==== Networking
+//CNV-10590 - This issue can be removed for 4.15 when CNV-13680 RN is added.
```

Comment:
@sjhala-ccs why is this commented out and not removed? (really asking, no idea)

Reply:
Was planning to remove it after I added in the release note for CNV-13680. Sorry about the confusion.